index.html

```diff
@@ -33,6 +33,274 @@
+              <ol>
+                <li>
+                  <p>Optionally, due to platform limitations, [=reject=]
+                    <var>p</var> with a new {{DOMException}} object whose {{DOMException/name}} attribute has the
+                    value {{NotAllowedError}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    Enumerate all <a data-cite="screen-capture#dfn-monitor">monitors</a>,
+                    resulting in a set of media to capture.
+                  </p>
+                  <p>
+                    Each provided media MUST include precisely one video track.
+                    Once selected, the source of each {{MediaDevices/ScreenCaptureMediaStreamTrack}} MUST NOT change.
+                  </p>
+                  <p>Each provided media MUST not include any audio track.</p>
+                  <p>For each device that is sourcing the selected medias, using
+                    a stable and private id for the device, <var>deviceId</var>,
+                    set [[\devicesLiveMap]]<var>[deviceId]</var> to
+                    <code>true</code>, if it isn’t already <code>true</code>,
+                    and set the
+                    [[\devicesAccessibleMap]]<var>[deviceId]</var> to
+                    <code>true</code>, if it isn’t already
+                    <code>true</code>.
+                  </p>
+                  <p>If a hardware error
+                    such as an OS/program/webpage lock prevents access to at least one device,
+                    <a>reject</a> <var>p</var> with a new
+                    {{DOMException}} object whose
+                    {{DOMException/name}} attribute has the value
+                    {{NotReadableError}} and abort these steps.
+                  </p>
+                  <p>If device access fails for
+                    any reason other than those listed above, <a>reject</a>
+                    <var>p</var> with a new {{DOMException}}
+                    object whose {{DOMException/name}} attribute has the
+                    value {{AbortError}} and abort these steps.
+                  </p>
+                </li>
+                <li>
+                  <p>Let <var>streams</var> be the
+                    list of {{MediaStream}} objects for which the permission was granted.</p>
+                </li>
+                <li>
+                  <p><a>Resolve</a> <var>p</var> with <var>streams</var> and
+                    abort these steps.</p>
+                </li>
+              </ol>
+            </li>
+            <li>
+              <p>Return <var>p</var>.</p>
+            </li>
+          </ol>
+          <p>
+            The <a href="#dfn-user-agent">user agent</a> MUST return a list of {{MediaStream}}
+            objects, where each stream belongs to exactly one of the
+            <a data-cite="screen-capture#dfn-monitor">monitors</a> attached to the device.
```

Comment:
Is that not a corollary of the algorithm?

Reply:
true. only had it here to make it explicit. removed.